aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/DeviceFarmServer.java

```diff
@@ -136,6 +140,17 @@ public void uploadApks(final String variantName, final File testPackage, final F
 
         final ScheduleRunResult response = api.scheduleRun(request);
 
+        if (extension.isWait()) {
+            DeviceFarmResultPoller poller = new DeviceFarmResultPoller(extension, logger, api, utils);
+            try {
+                Run completedRun = poller.pollTestRunForArn(response.getRun().getArn());
```

Comment:
Great PR, this would be an awesome addition.  
Although completedRun doesn't get used, maybe pollTestForRunForArn should be a void?
How is the CI going to know this was successful?

Reply:
@danPudwellUK 
That's part of the second PR pending on this one where we'll take the completedRun and consume the results into a given output (so far implemented only JUnit) but the ask was to split into 2 MRs so this is the first.

Reply:
thanks @andreas-marschke 
hopefully gets merged and released soon!